### ISSUE TITLE
[Feature required] Duplicate folder

### ISSUE THREAD

**Issue by zaaack** [facebook/nuclide#1048]
I think it's an important feature in original TreeView.

**Comment by psamim** [comment]
Is it removed from TreeView in Nuclide?

**Comment by zaaack** [comment]
@psamim Yes, nuclide's TreeView can only duplicate file, although it's context menu is much clear than original with a bunch of plugins, but is missing some important features, like duplicate folder, open with default app. BTW, I have disabled nuclide and using atom-ide-ui + linter, because linter-rust cannot work with nuclide's diagnostics ui, which atom-ide-ui can easily disabled.

**Comment by bradennapier** [comment]
wtf?  No duplicate or copy?  So you have to go find the finder, open it, go to things, deal with all that just to chang ethe folder structure of your project and/or duplicate?

This makes nuclide a useless product. 

Everything else is so amazing to have but this is just crazy not to have.  It's the basic essentials of file handling.......?
